guides/source/action_controller_overview.md

```diff
@@ -1174,6 +1174,9 @@ end
 
 WARNING: You shouldn't do `rescue_from Exception` or `rescue_from StandardError` unless you have a particular reason as it will cause serious side-effects (e.g. you won't be able to see exception details and tracebacks during development).
 
+NOTE: On production environment all `ActiveRecord::RecordNotFound` renders the
```

Comment:
Would you mind changing the wording of the sentence to the following? Makes the sentence a little easier to understand: When running in the production environment, all `ActiveRecord::RecordNotFound` errors render the 404 error page.